server/routers/viewer.tsx

```diff
@@ -295,6 +295,7 @@ const loggedInViewerRouter = createProtectedRouter()
             AND: [
               { NOT: { status: { equals: BookingStatus.CANCELLED } } },
               { NOT: { status: { equals: BookingStatus.REJECTED } } },
+              { NOT: { confirmed: { equals: false } } }, // hide unconfirmed bookings https://app.clickup.com/t/27pjd5c
```

Comment:
Are paid:true bookings always marked as confirmed?

Reply:
@zach-at-the-skills add back `{ NOT: { paid: { equals: false } } }` in addition to `{ NOT: { confirmed: { equals: false } } },` to cover all use-cases